src/utils/domPolyfill.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Polyfills the .closest method for ELement
+ * Currently being used for scrollIntoView
+ * For reference: https://github.com/zloirock/core-js/issues/317
+ *
+ * @param {HTMLElement|null} s - the dom string that is being checked against
+ * @return {HTMLElement|null} returns either the parent element that was found or null
+ */
+window.Element.prototype.closest = function closest(s) {
```

Comment:
For consistency, I would just drop window. part, since at other places you are referencing Element directly. Or change other places to use window. too.